tasks.py

```diff
@@ -0,0 +1,47 @@
+import sys
+from invoke import task
+
+
+@task
+def benchmark(c):
+    """CI benchmark"""
+    c.run(
+        "pytest --benchmark-only --benchmark-group-by=param:h5path --benchmark-columns=median,iqr,min,max"
```

Comment:
Are options needed here since they are already in `pyproject.toml`?

https://github.com/silx-kit/h5grove/blob/8867c17cfeb72301d5944e1ad5698a03efe9212e/pyproject.toml#L7

Reply:
Good call, I didn't see the options in `pyproject.toml` !